tests/nlu/training_data/converters/test_watson_nlu_json_to_yaml_converter.py

```diff
@@ -0,0 +1,36 @@
+import os
+from pathlib import Path
+
+import pytest
+
+from rasa.nlu.training_data.converters.watson_nlu_json_to_yaml_converter import (
+    WatsonTrainingDataConverter,
+)
+from rasa.shared.nlu.training_data.loading import load_data
+
+
+def test_watson_data():
+    td = load_data("data/examples/watson/demo_watson_v2.json")
+    assert not td.is_empty()
+    assert len(td.entity_examples) == 117
+    assert len(td.intent_examples) == 309
+
+
+def test_filter():
+    source = Path("data/examples/watson/demo_watson_v2.json")
+    filter = WatsonTrainingDataConverter().filter(source)
+    if filter:
+        assert True
```

Comment:
can we also add some tests where `filter` is `False`?

Reply:
yeah, i will add it